# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/InterceptedHttpChannel.h
# Commit: aaf5c5213d8d
# Full Hash: aaf5c5213d8db7c9344e8517ae934c55f937e07e
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2020-10-08 09:49:50
# Regressor Bug: 1665062
# File Overlap Count: 1
# Description:
#   Bug 1665062: HTTPS-Only: Upgraded website creating HTTP auth prompt gets interrupted by error-page r=necko-reviewers,dragana,JulianWels
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91908
# ==============================================================================

diff -r e4b87a2e0f2c -r aaf5c5213d8d netwerk/protocol/http/InterceptedHttpChannel.h
--- a/netwerk/protocol/http/InterceptedHttpChannel.h	Tue Oct 06 12:51:16 2020 +0000
+++ b/netwerk/protocol/http/InterceptedHttpChannel.h	Wed Oct 07 11:47:07 2020 +0000
@@ -160,6 +160,9 @@
   SetupFallbackChannel(const char* aFallbackKey) override;
 
   NS_IMETHOD
+  GetIsAuthChannel(bool* aIsAuthChannel) override;
+
+  NS_IMETHOD
   SetPriority(int32_t aPriority) override;
 
   NS_IMETHOD